### ISSUE TITLE
transform clash

### ISSUE THREAD

**Issue by vidstige** [joshmarinacci/node-pureimage#68]
### Expected Behaviour

Call `context.transform()` changes the transform.

### Actual Behaviour

Error: `context.transform is not a function`

### Steps To Reproduce

The following simple snippet does not work.
```js
require('pureimage').make(2, 2).getContext("2d").transform(1, 1, 0, 0, 1, 1);
```

### Platform
**OS:** MacOS
**Node Version:** 
**NPM Version:**
**PureImage Version:**

### Any Additional Info
It seems the `transform` key is some kind of object. There _is_ a `transform.setTransform` that does the same things as `transform` in the browser. Perhaps this object can be renamed to `_transform`, and then we just assign `this.transform = this._transform.setTransform`.

**Comment by joshmarinacci** [comment]
This fix is released in 0.3.14

**Comment by vidstige** [comment]
sweet
